### ISSUE TITLE
is the mailing list down? 

### ISSUE THREAD

**Issue by 2ros0** [PointCloudLibrary/pcl#2415]
Apologies for posting a question here but the mailing list seems to be down. The most recent post is from July. Is it just the delay in accepting new questions? Thanks

**Comment by SergioRAgostinho** [comment]
Yeah. Sorry for that. @rbrusu is undergoing a migration of the mailing list to Google groups.

**Comment by kervel** [comment]
any news wrt the mailing list ? is there already a new mailing list location ?
thanks
Frank

**Comment by SergioRAgostinho** [comment]
I halted it to finish the 1.9.0. It's the next thing on my todo list. 

In terms of status: in theory the approach to accomplish what is needed is already devised.  The parsing of the old mailing list is complete, but the actual migration (uploading the email archive and users) to Google Groups posed some challenges.

**Comment by leonardokidd** [comment]
Is there any update about the mailing list? I was not able to post the question in the beginning of the May, and I'm waiting for the response after reposting. If mailing list still requires some time to back on line, can we temporarily use github for questions? Thanks in advance.

**Comment by SergioRAgostinho** [comment]
Hey guys. So before starting the sprint for 1.10 me and Sergey had a brief chat about this. Answering directly your question, no progress was made on the ML. Still, we noticed that even if we allocate time to fix things there, there is still an important aspect missing which is people to provide responses or support. Time is finite for everyone and as volunteers on maintaining this project, we do agree that our time is better spent actually coding, reviewing PRs and moving the library forward vs going through support questions.

With this in mind I suggested to move everything to Stack Overflow and simply recommend the users to ask their questions there, under the tag "point-cloud-library". 

Getting things on SO has some interesting benefits:
- way bigger community
- general C++/CMake questions can be answered by a broader audience
- question filtering is crowdsourced: SO forces users to expose their problems in a proper manner, otherwise they are just down voted or closed.
- analytics: at the end of the day, we can look at the most upvoted questions and see what is more important to be improved in our tutorials and FAQs
- public acknowledgement: when someone asks a good question or answers a question, they get those SO badges which "kind of represent their overall credibility and expertise". Some people might find that a compelling reason to be more active in helping others.

Things which might go wrong from using SO:
- The mailing list format in undoubtedly more flexible in the interaction.
- We lose a channel for announcements. We'll need to do it through GitHub and through the Website (once that one is fixed/updated).

Ultimately we're trying to be realistic about what we can do. PCL grew very quickly in its early days, it managed to raise a lot of attention, the code base grew out of control and the community followed alongside. In this day and age, the community is way larger than the people it actually has to maintain it effectively. Hence my belief that we need to tailor our workload to the current reality.

**Comment by leonardokidd** [comment]
Thank you very much. Appreciate for the answer.

**Comment by larshg** [comment]
Can we close this one @kunaltyagi ?
